include/spirv-tools/optimizer.hpp

```diff
@@ -432,6 +432,11 @@ Optimizer::PassToken CreateMergeReturnPass();
 // This pass will look for instructions in the same basic block that compute the
 // same value, and remove the redundant ones.
 Optimizer::PassToken CreateLocalRedundancyEliminationPass();
+
+// Create global value numbering pass.
+// This pass will look for instructions in the same function that compute the
+// same value, and remove the redundant ones.
```

Comment:
Clarify that this is done in cases where there is **total** redundancy: "compute the same value through all execution paths", maybe?